src/services/structures/structures.hooks.js

```diff
@@ -121,6 +122,8 @@ module.exports = {
           Object.assign(structure, { dernierCoselec: utils.getCoselec(structure) });
         });
 
+        context.result.data = context.result.data.filter(structure => !isStructureDuplicate(structure));
```

Comment:
Faudrait tout réécrite mais 
```suggestion
        context.result.data = context.result.data.filter(structure => StructureIsNotDuplicate(structure));
```
pour ne pas avoir de condition négative ?

Reply:
Il vaut mieux éviter les fonction négatives pour conserver le même sens de raisonnement, l'opératieur ! est la pour faire la convertion